lib/phases/reboot.js

```diff
@@ -63,6 +63,7 @@ Reboot.prototype.reboot = function() {
       return phase._restart();
     })
     .then(function(time) {
+      phase.setup();
```

Comment:
For the `adb forward`, I don't mind it being moved, but it has to be done *after* the actual restart since rebooting will clear the forward. So I'm not sure there is a real benefit to moving it really.